### PR TITLE
Add GitHub lint and test actions

### PR DESCRIPTION
**Pull request by efoxepstein**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2)

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Lint
```
Comment (EvgSkv) [inline .github/workflows/lint.yml:1]: Does this file need the copyright header?

Comment (efoxepstein) [reply .github/workflows/lint.yml:1]: Nope (it doesn't in other repos)

**Review by EvgSkv — APPROVED** [review]

**Review by oliver-amzn — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2)* status: 0 of 4 files reviewed, 2 unresolved discussions (waiting on @efoxepstein, @EvgSkv, and @mpatel24)

___
*[.github/workflows/build-test.yml, line 33 at r3](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2#-Mfixtzt4DOZtuEx0ORg:-Mfixtzt4DOZtuEx0ORh:b-umh0jy) ([raw file](https://github.com/world-federation-of-advertisers/virtual-people-training/blob/dc45c96d95637c9bee10e50a7120d558eb8cb0b1/.github/workflows/build-test.yml#L33)):*
> ```YAML
>         uses: actions/checkout@v2
> 
>       - name: Set execroot hash
> ```

Any reason we're not using `world-federation-of-advertisers/actions/bazel-build-test@v1.3.0` here (like in the xmm repo) and isntead do all this?

**Review by efoxepstein — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2)* status: 0 of 4 files reviewed, 2 unresolved discussions (waiting on @EvgSkv, @mpatel24, and @oliver-amzn)

___
*[.github/workflows/build-test.yml, line 33 at r3](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2#-Mfixtzt4DOZtuEx0ORg:-Mfj6TJY5vlk6Zurnt08:bb6un1j) ([raw file](https://github.com/world-federation-of-advertisers/virtual-people-training/blob/dc45c96d95637c9bee10e50a7120d558eb8cb0b1/.github/workflows/build-test.yml#L33)):*
<details><summary><i>Previously, oliver-amzn (Oliver) wrote…</i></summary><blockquote>

Any reason we're not using `world-federation-of-advertisers/actions/bazel-build-test@v1.3.0` here (like in the xmm repo) and isntead do all this? 
</blockquote></details>

Done

**Review by oliver-amzn — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2)* status: 0 of 4 files reviewed, 1 unresolved discussion (waiting on @EvgSkv and @mpatel24)

___
*[.github/workflows/build-test.yml, line 33 at r3](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-training/2#-Mfixtzt4DOZtuEx0ORg:-MfjGGfJ2T0znPLhDzov:b44uc2) ([raw file](https://github.com/world-federation-of-advertisers/virtual-people-training/blob/dc45c96d95637c9bee10e50a7120d558eb8cb0b1/.github/workflows/build-test.yml#L33)):*
<details><summary><i>Previously, efoxepstein (Eli Fox-Epstein) wrote…</i></summary><blockquote>

Done
</blockquote></details>

🎉

**Pull request merged**